### DISCUSSION TITLE
Recording streams 24/7 - automatic MP4 file splitting?

### DISCUSSION THREAD

**Discussion by mstepuch** [ant-media/Ant-Media-Server#5527] [Q&A]
Hello,

I use AMS community ed. to restream video from three IP cams to YT and other social media.
Thes are live nature cams which run 24/7
Additionaly I need to record this video streams for future analysis.

I run AMS on VPS where i have limited drive space (about 130 Gb), so I use rclone to periodicaly move all MP4 files to Google Drive.
It works perfect. Tranfferring files and deleting them.

The only problem is AMS does not split files. It saves them until their size is so large that the free space on the disk runs out.
It would be enough if the files were saved, for example, every 10Gb, then rclone could move them and free up space on the disk.

Any idea how to do it?

**Comment by USAMAWIZARD** [comment]
@mstepuch  there is no direct way to  specify the file size for now .  one way of doing that would be start and stop the recording from the dash board or automate the whole process through apis , 

Check out [this Rest API](https://antmedia.io/rest/#/BroadcastRestService/enableRecording) for starting and stopping the recording/

**Comment by mstepuch** [comment]
Thank You for fast response. 
I need to learn how to use Rest API.

Can I use CRON to start/stop record using Rest ?

**Reply by USAMAWIZARD** [reply]
@mstepuch yes you can use crone jobs to call the rest apis ,  why don't you use S3 instead in that way you don't need to upload to drive or worry about the disk size , as the files are automatically deleted once they are uploaded to s3 bucket

**Comment by mstepuch** [comment]
So i have to learn how tu use Rest API
I know I can use S3, but I allready had GDrive when I started using AMS.
I'll try to swich to S3

Tahnk You

**Reply by USAMAWIZARD** [reply]
here is the [documentation](https://antmedia.io/docs/guides/developer-sdk-and-api/rest-api-guide/) for REST API 
[S3 docs](https://antmedia.io/docs/guides/recording-live-streams/s3-integration-http-forwarding/)

**Reply by mstepuch** [reply]
Thanks

**Reply by dmtan90** [reply]
@mstepuch Another way, you can set the duration (seconds) for each MP4 file by settings.streamFetcherRestartPeriod in the property file

**Reply by mstepuch** [reply]
Wow, this could be better solution which I am looking for.
What propperty file should I edit?

**Reply by USAMAWIZARD** [reply]
@dmtan90 Thanks for the info I did not knew that

**Reply by lastpeony** [reply]
Go to your apps directory
ant-media-server/webapps/WebRTCAppEE/WEB-INF
edit red5-web.properties file

**Comment by mstepuch** [comment]
OK. So i set this parameter to 14400 (4 hours)
Restarted service and waiting :-)
